src/views/queries.tsx

```diff
@@ -0,0 +1,52 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { Page } from './common'
+
+@singleton()
+export default class QueriesTemplates {
+  constructor() {}
+  public chooseQueryPage = () => {
+    return (
+      <Page
+        title="Veritable - Queries"
+        heading="Queries"
+        headerLinks={[{ name: 'Queries', url: '/queries' }]}
+        stylesheets={['new-invite.css']}
+      >
+        <div class="main connections" hx-get="/queries"></div>
+        <div>
+          <h1 class="query-page-header">Queries</h1>
+          <hr class="divider"></hr>
+        </div>
+        <div class="query-container">
+          <div class="query-item">
```

Comment:
These should all be links right? Even if they just go to `#`. This will also mean you need a `focus-visible` state